modules/parquet/src/parquet-wasm-loader.ts

```diff
@@ -34,7 +34,7 @@ export const ParquetWasmWorkerLoader: Loader<ArrowTable, never, ParquetWasmLoade
   options: {
     parquet: {
       type: 'arrow-table',
-      wasmUrl: 'https://unpkg.com/parquet-wasm@0.3.1/esm2/arrow1_bg.wasm'
```

Comment:
It's great that `esm` is working; the original `esm2` endpoint was an undocumented hack that just stripped the `import.meta.url` line out of the generated JS bundle with `sed`, which we seemed to need for bundling at the time.